### ISSUE TITLE
ASN1 broken?

### ISSUE THREAD

**Issue by J08nY** [SecurityInnovation/PGPy#209]
This is a traceback I get when verifying a message with an ECC key (P-256), happens atleast on e183c68 as my dev branch is based on that, and it happens in there. But AFAIK this was introduced by an upstream change in pyasn1, so it might have broken even other PGPy releases, for someone that updated pyasn1.

```python
----------------------------------------------------------------------
  File "/home/johny/psf/mailman/PGPy/pgpy/pgp.py", line 2301, in verify
    verified = self._key.verify(sig.hashdata(subj), sig.__sig__, getattr(hashes, sig.hash_algorithm.name)())
  File "/home/johny/psf/mailman/PGPy/pgpy/pgp.py", line 94, in __sig__
    return self._signature.signature.__sig__()
  File "/home/johny/psf/mailman/PGPy/pgpy/packet/fields.py", line 273, in __sig__
    return encoder.encode(seq)
  File "/home/johny/psf/mailman/mailman-pgp/.tox/py36-dev-cov/lib/python3.6/site-packages/pyasn1/codec/der/encoder.py", line 56, in __call__
    return encoder.Encoder.__call__(self, value, defMode, maxChunkSize, ifNotEmpty=ifNotEmpty)
  File "/home/johny/psf/mailman/mailman-pgp/.tox/py36-dev-cov/lib/python3.6/site-packages/pyasn1/codec/cer/encoder.py", line 203, in __call__
    return encoder.Encoder.__call__(self, value, defMode, maxChunkSize, ifNotEmpty)
  File "/home/johny/psf/mailman/mailman-pgp/.tox/py36-dev-cov/lib/python3.6/site-packages/pyasn1/codec/ber/encoder.py", line 483, in __call__
    self, value, defMode, maxChunkSize, ifNotEmpty=ifNotEmpty
  File "/home/johny/psf/mailman/mailman-pgp/.tox/py36-dev-cov/lib/python3.6/site-packages/pyasn1/codec/ber/encoder.py", line 61, in encode
    encodeFun, value, defMode, maxChunkSize, ifNotEmpty=ifNotEmpty
  File "/home/johny/psf/mailman/mailman-pgp/.tox/py36-dev-cov/lib/python3.6/site-packages/pyasn1/codec/cer/encoder.py", line 151, in encodeValue
    substrate = encodeFun(value[idx], defMode, maxChunkSize, namedTypes[idx].isOptional) + substrate
  File "/home/johny/psf/mailman/mailman-pgp/.tox/py36-dev-cov/lib/python3.6/site-packages/pyasn1/type/namedtype.py", line 152, in __getitem__
    return self.__namedTypes[idx]
IndexError: tuple index out of range

----------------------------------------------------------------------
```

```
pyasn1==0.3.2
```

**Comment by J08nY** [comment]
This is fixed in current master by: afdde8c40e279c51d274049a81518b401fff2b11, getting a new release out there is quite important, as it seems 0.4.1 is broken too (by pyasn1 update to 0.3.2).

**Comment by J08nY** [comment]
[New release](https://pypi.org/project/PGPy/) is up. Issue no longer relevant.
